lib/api/etas_api.rb

```diff
@@ -0,0 +1,59 @@
+require "cluster"
+require "clusterable/ht_item"
+require "services"
+require "sinatra"
+
+class EtasAPI < Sinatra::Base
+  set :show_exceptions, ENV["show_sinatra_exceptions"] || false
+  no_matching_data_error = {"application_error" => "no matching data"}.to_json
+
+  get "/ping" do
+    "pong"
+  end
+
+  get "/etas/item_access" do
+    # ArgumentError if missing organization / ht_id.
+    validate_params(params, ["organization", "ht_id"])
+
+    organization = params["organization"]
+    ht_id = params["ht_id"]
+
+    ht_item = Clusterable::HtItem.find(item_id: ht_id)
+    cluster = ht_item.cluster
+
+    return_doc = {
+      "lock_id" => ht_item.ocns.sort.join(","),
```

Comment:
This won't always correspond to the lock id as computed now, maybe we should just call it "ocns" and return it as a JSON array. I also wonder if we should return format?

Reply:
Changed `lock_id` (string) ==> `ocns` (array), added `format` (value of `cluster.format` for the cluster of the ht_item).